app/models/labels/base.rb

```diff
@@ -24,6 +24,10 @@ def qc_attributes
     []
   end
 
+  def sprint_attributes
```

Comment:
You can also use alias_method here. i.e. alias_method :sprint_attributes, :attributes

Reply:
That would be nice - it seems to not like it because the attributes method is defined in the inheriting class so its getting undefined_method